src/ipc/Inquirer.cc

```diff
@@ -120,18 +120,18 @@ Ipc::Inquirer::doneAll() const
 void
 Ipc::Inquirer::handleException(const std::exception& e)
 {
-    debugs(54, 3, HERE << e.what());
+    debugs(54, 3, e.what());
     mustStop("exception");
 }
 
 void
 Ipc::Inquirer::callException(const std::exception& e)
 {
-    debugs(54, 3, HERE);
+    debugs(54, 3, "");
     try {
         handleException(e);
     } catch (const std::exception& ex) {
-        debugs(54, DBG_CRITICAL, HERE << ex.what());
+        debugs(54, DBG_CRITICAL, ex.what());
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.